.github/workflows/main.yml

```diff
@@ -82,6 +82,20 @@ jobs:
           FAUCET_AUTH_TOKEN: ${{ secrets.FAUCET_AUTH_TOKEN }}
           FAUCET_TOKEN_LIMIT: 100
 
+      - name: API tests
+        run: yarn test:api --ci --coverage --maxWorkers=2
+        env:
+          API_URL: http://${{ env.voconed_hostport }}/v2
+          BLINDCSP_URL: http://${{ env.blindcsp_hostport }}/v1
+          BLINDCSP_PRIVKEY: f6e530882cde11e2050989d04c3fc523412b2fc2723c0a6155932a62ffafc2b6
```

Comment:
I guess this is for development only, but doesn't look quite good to have a private key exposed. If it's not problematic at all, np; otherwise this should be a github secret.

Reply:
I think it's not problematic, but I just pasted the @altergui code. I agree it's not nice to have the private key exposed here, even if it's just for testing purposes.

Reply:
this key is also hardcoded in
https://github.com/vocdoni/vocdoni-sdk/blob/75d1fe938aa182bba1fbd02d42b03ff58f36af61/test/integration/util/docker-compose.yml#L35

and they must be equal for the (local) test to work.

i guess to be perfectly clear that it's not a secret and it's used nowhere outside of this repo, we could replace both lines with 

`deadbeef00000000000000000000000000000000000000000000000000000000`

Reply:
but then the following comment needs updating and most importantly the BLINDCSP_PUBKEY line

Reply:
(i agree committing a privkey doesn't look good)

Reply:
That `deadbeef` key looks like a better candidate for development/testing purposes 😄